api/spec/v1.yaml

```diff
@@ -2049,6 +2058,15 @@ components:
         success:
           type: boolean
           description: Whether this transaction successfully executed.
+        code:
+          type: integer
+          description: The status code of a failed call.
+        module:
+          type: string
+          description: The module of a failed call.
+        message:
+          type: string
+          description: The message of a failed call.
```

Comment:
Should we prefix these fields with `err`, eg `errCode`? I feel like it might be confusing otherwise, despite the field description.

Also, can we use `transaction` instead of `call`? Are the two the same?

Reply:
+1 to prefix, or even better, I'd go for a sub-struct
```yaml
error:
  $ref: '#/components/schemas/TxErrorDetails'

...
TxErrorDetails:
  description: Details about why and how the transaction failed.
  type: object
  properties:
    module: ...
    code: ...
    message: ...
```

Also +1 to call->transaction; let's continue in the other thread above.